lib/marked.js

```diff
@@ -647,6 +647,7 @@ function InlineLexer(links, options) {
   } else if (this.options.gfm) {
     if (this.options.breaks) {
       this.rules = inline.breaks;
+      this.rules.url = new RegExp(this.rules.url.source, "i");
```

Comment:
The lint step is failing because we expect single quotes: `new RegExp(this.rules.url.source, 'i');`

Reply:
Thanks!